.github/workflows/build-images.yaml

```diff
@@ -114,8 +117,14 @@ jobs:
         id: add-latest-tag
         run: |
           echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Generate Catalog Content
-        run: make catalog
+      - name: Run make catalog
```

Comment:
pretty much the same as above. I would leave the changes you propose as they are.